### ISSUE TITLE
NO Issue, Add Graph of Ampere

### ISSUE THREAD

**Issue by atc1441** [najnesnaj/platformio-stm32-ampere#1]
After looking trough the code again i am thinking it would be quite easy to add a graph of the current measurements, 

As the Current is measure every 100ms but only shown every 3 Seconds we have plenty of measurements that are just trown away.

Just an idea :D

**Comment by najnesnaj** [comment]
As the measurements are sent to serial port, one could use the graph serial monitor of the Arduino Software (IDE) to view the sent data, or it can be read by Processing ...

**Comment by atc1441** [comment]
You Are Right, had to adjust a bit the delay time but it looks nice and is more precise on the serial plotter.
Not worth the work :D

and in reality the measurements also take about 200ms so not really 100ms overall.

![SerialPlotter](https://user-images.githubusercontent.com/1102694/62650461-38ef2200-b957-11e9-9d44-a7032396b8eb.jpg)
